sp/src/game/server/ez2/npc_flyingpredator.h

```diff
@@ -37,6 +37,10 @@ class CNPC_FlyingPredator : public CNPC_BasePredator
 
 	virtual int RangeAttack1Conditions( float flDot, float flDist );
 
+	// Used by base LOS functions
+	virtual float	InnateRange1MinRange( void ) { return 64.0f; }
```

Comment:
Why does innate range attack 1 need a minimum range? Why can't they dive bomb from any range?

Reply:
This function is used by base LOS tasks such as `TASK_GET_PATH_TO_ENEMY_LOS` or `TASK_GET_FLANK_ARC_PATH_TO_ENEMY_LOS`, which are used by the new stukabat schedules. This does not actually prevent stukabats from attacking at this range.